src/components/Dashboard/Charts/SentEmailCampaigns/SentEmailCampaigns.tsx

```diff
@@ -79,32 +122,82 @@ const SentEmailCampaigns = ({ isMailsStatutory = false }) => {
           ),
         }}
       />
-      {drafts.length > 0 && (
-        <Box>
-          <Typography variant="h6" sx={{ mb: 2 }}>
-            Brouillions
-          </Typography>
-          <Grid container spacing={3}>
-            {drafts.map(message => (
-              <Grid item key={message.id} xs={12} sm={6} md={4} xl={3} data-cy="email-campaign-card">
-                <MessageCard message={message} onPopoverOpen={onPopoverOpen} />
-              </Grid>
-            ))}
-          </Grid>
+      {!isFetching && messages && messages.length > 0 && (
+        <Box display="flex" flexDirection="column" gap={2}>
+          <MessageList messages={messages} onPopoverOpen={props.onPopoverOpen} />
         </Box>
       )}
-      <Box>
-        <SentEmailCampaignsTitle isMailsStatutory={isMailsStatutory} />
-        <InfiniteScroll dataLength={sents.length} next={fetchNextPage} hasMore={!!hasNextPage} loader={<Loader />}>
-          <Grid container spacing={3}>
-            {sents.map(message => (
-              <Grid item key={message.id} xs={12} sm={6} md={4} xl={3} data-cy="email-campaign-card">
-                <MessageCard message={message} onPopoverOpen={onPopoverOpen} />
-              </Grid>
-            ))}
-          </Grid>
-        </InfiniteScroll>
-      </Box>
+
+      {isFetching && <MessageLoader />}
+    </Stack>
+  )
+}
+
+const MessageDraftSection = ({ onPopoverOpen }: { onPopoverOpen: OnPopoverOpen }) => {
+  const queryKeyDrafts = useScopedQueryKey(['draft-campaigns', { feature: 'Dashboard', view: 'SentEmailCampaigns' }])
+  const { handleError } = useErrorHandler()
+  const {
+    data = [],
+    isInitialLoading,
+    isFetched,
+  } = useQuery({
+    queryFn: () => getMessages({ status: 'draft', pagination: false }),
+    queryKey: queryKeyDrafts,
+    onError: handleError,
+  })
+
+  return (
+    <Box display="flex" flexDirection="column" gap={2}>
+      <Typography variant="h6">Brouillions</Typography>
+      <MessageList messages={data} onPopoverOpen={onPopoverOpen} />
+      {isInitialLoading && <MessageLoader />}
+      {isFetched && data.length === 0 && <EmptyStatus text="0 brouillons" />}
+    </Box>
+  )
+}
+
+const MessageSentSection = (props: { onPopoverOpen: OnPopoverOpen; isMailsStatutory: boolean }) => {
+  const queryKey = useScopedQueryKey(['paginated-campaigns', { feature: 'Dashboard', view: 'SentEmailCampaigns' }])
+  const { handleError } = useErrorHandler()
+  const { data, fetchNextPage, hasNextPage, isFetched, isInitialLoading } = useInfiniteQuery({
+    queryKey,
+    queryFn: ({ pageParam: page = 1 }) =>
+      getMessages({ statutory: props.isMailsStatutory, pagination: true, page, page_size: 20, status: 'sent' }),
```

Comment:
on peut retirer le param pagination car elle est activée par défaut sur l'API 
```suggestion
      getMessages({ statutory: props.isMailsStatutory, page, page_size: 20, status: 'sent' }),
```